### ISSUE TITLE
delete this repository

### ISSUE THREAD

**Issue by iantaylor-NOAA** [r4ss/SSutils#2]
I'm planning on delete this repository as a step toward moving r4ss into nmfs-stock-synthesis as discussed long ago in https://github.com/r4ss/r4ss/issues/815 but now rejuvenated as a plan.

@k-doering-NOAA, the only thing in this repo that hasn't been migrated to the r4ss repo is the [run_SS_models_parallel.R](https://github.com/r4ss/SSutils/blob/master/R/run_SS_models_parallel.R) file which I've just attached to this discussion that references it: https://github.com/r4ss/r4ss/issues/775#issuecomment-1424684268.

Recent progress toward parallelization in discussed in https://github.com/r4ss/r4ss/issues/775 and https://github.com/r4ss/r4ss/pull/870 has been focused on allowing the user to control the furrr commands outside of the r4ss function, rather than within it.

**Comment by k-doering-NOAA** [comment]
@iantaylor-NOAA, I'm fine with it being deleted! I used to use this parallelized workflow for running local tests, but as long as @e-gugliotti-NOAA isn't using it (which I don't think she is), I think we can let it go given progress made in the last 4 years (aside: I can't believe my last commit on the file was 4 years ago, wow!)

**Comment by e-gugliotti-NOAA** [comment]
Yeah, the parallelization isn't this workflow didn't ever work for me, so I haven't been using it. I have a different parallel workflow set up in my simulations work (that one day I will finish lol) that works for me.

**Comment by k-doering-NOAA** [comment]
perfect - It's probably not surprising it doesn't work, given it is 4 years old 😆
